### PR TITLE
Update install from source instructions

### PR DESCRIPTION
**Pull request by Justman10000**

### CODE REVIEW

File: docs/getting-started/installation/install-redis-from-source.md [hunk 1]
```diff
@@ -30,12 +36,13 @@ make
 
 If the compile succeeds, you'll find several Redis binaries in the `src` directory, including:
 
-* **redis-server**: the Redis Server itself
-* **redis-cli** is the command line interface utility to talk with Redis.
+* **redis-server**: The Redis Server itself
+* **redis-cli** : The command line interface utility to talk with Redis.
+* **redis-benchmark**: The command line tool for benchmarks
```
Comment (enjoy-binbin) [inline docs/getting-started/installation/install-redis-from-source.md:41]: i like this, let's also mention:
- redis-sentinel
- redis-check-rdb
- redis-check-aof

File: docs/getting-started/installation/install-redis-from-source.md [hunk 2]
```diff
@@ -30,12 +36,13 @@ make
 
 If the compile succeeds, you'll find several Redis binaries in the `src` directory, including:
 
-* **redis-server**: the Redis Server itself
-* **redis-cli** is the command line interface utility to talk with Redis.
+* **redis-server**: The Redis Server itself
+* **redis-cli** : The command line interface utility to talk with Redis.
```
Comment (enjoy-binbin) [inline docs/getting-started/installation/install-redis-from-source.md:40]:
```suggestion
* **redis-cli**: The command line interface utility to talk with Redis.
```

File: docs/getting-started/installation/install-redis-from-source.md [hunk 3]
```diff
@@ -18,6 +18,12 @@ To obtain the source files for the latest stable version of Redis from the Redis
 wget https://download.redis.io/redis-stable.tar.gz
 {{< / highlight >}}
 
+## Install needed dependencies
+
+{{< highlight bash >}}
+apt install gcc -y
```
Comment (itamarhaber) [inline docs/getting-started/installation/install-redis-from-source.md:24]: I usually do `apt install build-essential tcl8.5` - this tales care of most deps and also prepares for running `make test`.

File: docs/getting-started/installation/install-redis-from-source.md [hunk 4]
```diff
@@ -47,6 +54,14 @@ Once installed, you can start Redis by running
 redis-server
 {{< / highlight >}}
 
-If successful, you'll see the startup logs for Redis, and Redis will be running in the foreground.
+If successful, you'll see the startup logs for Redis. Redis will be running in the foreground.
+
+Or
+
+{{< highlight bash  >}}
+screen -S Redis redis-server
```
Comment (itamarhaber) [inline docs/getting-started/installation/install-redis-from-source.md:62]: `screen` also needs to be installed in this case. I suggest just:

```suggestion
./src/redis-server
```

**Comment by Justman10000** [comment]
@itamarhaber

**Comment by Justman10000** [comment]
@enjoy-binbin

**Comment by enjoy-binbin** [comment]
why adding the `install-redis-by-script`? we generally do not depend on external things (distrust)

**Comment by enjoy-binbin** [comment]
> My script is open source, you can see the entire code...

I know, but I don't think it is necessary.

**Comment by Justman10000** [comment]
> > My script is open source, you can see the entire code...
> 
> I know, but I don't think it is necessary.

How? I have specially written a script, what Redis completely set up, which you can also manage via the script! I have definitely not sat down at the script, only to read such sentences!

**Comment by Justman10000** [comment]
> > My script is open source, you can see the entire code...
> 
> i suggest removint it, and then maybe we can update this https://github.com/redis/redis-doc/pull/2014/files#r906949087
> 
> so that in my side, i can call help to merge this one

Like mine with update?

File: docs/getting-started/installation/install-redis-from-source.md [hunk 5]
```diff
@@ -30,23 +42,44 @@ make
 
 If the compile succeeds, you'll find several Redis binaries in the `src` directory, including:
 
-* **redis-server**: the Redis Server itself
-* **redis-cli** is the command line interface utility to talk with Redis.
+* **redis-server**: The Redis Server itself
+* **redis-cli**: The command line interface utility to talk with Redis.
+* **redis-benchmark**: The command line tool for benchmarks
```
Comment (enjoy-binbin) [inline docs/getting-started/installation/install-redis-from-source.md:47]: let's also mention 
redis-sentinel
redis-check-rdb
redis-check-aof

**Comment by enjoy-binbin** [comment]
my advice has been given, thank you for your contribution, maybe @itamarhaber can help with it (the rest part)

**Comment by Justman10000** [comment]
> my advice has been given, thank you for your contribution, maybe @itamarhaber can help with it (the rest part)

Like mine?

File: docs/getting-started/installation/install-redis-from-source.md (hunk 5 above)
Comment (Justman10000) [reply docs/getting-started/installation/install-redis-from-source.md:47]: Need the descriptions...

**Comment by nermiller** [comment]
@enjoy-binbin I don't think we use third-party repositories for any Redis docs. Also, this doc is undergoing major revisions. Ok to close for now?

**Comment by enjoy-binbin** [comment]
>  I don't think we use third-party repositories for any Redis docs. Also, this doc is undergoing major revisions. Ok to close for now?

yes, we should not use third-party repositories. undergoing major revisions, let's close it.

@Justman10000 thanks for the contribution, and sorry for the delay, feel free to refresh it

**Comment by Justman10000** [comment]
> > I don't think we use third-party repositories for any Redis docs. Also, this doc is undergoing major revisions. Ok to close for now?
> 
> yes, we should not use third-party repositories. undergoing major revisions, let's close it.
> 
> @Justman10000 thanks for the contribution, and sorry for the delay, feel free to refresh it

WHY NOT? Then regulate the shit yourself! No more support of any kind from me!

**Comment by enjoy-binbin** [comment]
we have explained our reasons, and i think i have made it clear.
i'm not targeting you (or anyone), if you have better opinions, you can continue to put forward
we all have the same common goal and that is to make it better

**Comment by Justman10000** [comment]
> we have explained our reasons, and i think i have made it clear. i'm not targeting you (or anyone), if you have better opinions, you can continue to put forward we all have the same common goal and that is to make it better

Well, I have submitted plenty of PRs in quite a few projects! Since one has times almost the face full, if loosely 80-85% of all PRs are closed! Some then even without justification...

And I would also take good care of the script, PRs are also welcome... So where is the problem? If the projects I write these scripts for don't accept them, what's the point of putting out there? After all, it has added value for the community....

**Pull request closed without merging**